src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/EncodeUrl.cs

```diff
@@ -1,10 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-using System;
 using System.Collections.Generic;
-using System.Text;
```

Comment:
This function should be added to BuiltinFunctionsCore and removed from PAClient's builtinfunctions, seems like it was missed in the initial migration

Reply:
I have [this PR out](https://msazure.visualstudio.com/OneAgile/_build/results?buildId=70286885&view=results), but due to some broken enum changes, I get to change PFx version. working on it.